ClientLib/src/main/java/com/o3dr/android/client/apis/FollowApi.java

```diff
@@ -46,14 +50,20 @@ private FollowApi(Drone drone) {
     /**
      * Enables follow-me if disabled.
      *
-     * @param followType follow-me mode to use.
+     * @param type follow-me mode to use.
+     * @param useExternal true if the API will expect locations from the client app.
      */
-    public void enableFollowMe(FollowType followType) {
+    public void enableFollowMe(FollowType type, boolean useExternal) {
```

Comment:
@kellyschrock as discussed, let's use an enum instead. This gives us more flexibility if we decide to introduce addition location providers in the future.